web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme

```diff
@@ -123,6 +123,23 @@ function genlyd_aarhus_preprocess_node__activity(&$variables) {
     ],
   ];
 
+  // Define subject and body variables for mailto link.
+  $subject =  \Drupal::translation()->translate('Here is a suggestion for an activity in a community in Aarhus');
+  $bodyTop = \Drupal::translation()->translate('Try to have a look at this activity in Aarhus.');
+  $bodyBottom = \Drupal::translation()->translate('Open this link to see the activity');
+  // Will be urlencoded to two new lines.
+  $newlines = '
+
+';
+  // Setup mailto link.
+  $mailto =
```

Comment:
You could put the parts into an array and use:

`string implode ( string $glue , array $pieces )`